src/test/java/apicurito/tests/steps/VerificationSteps/AssertAll.java

```diff
@@ -0,0 +1,19 @@
+package apicurito.tests.steps.VerificationSteps;
```

Comment:
change package name `apicurito.tests.steps.VerificationSteps` to lower case

Reply:
updated